process/closing-wg.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" >
+<title>How to Close a Working Group</title>
+<link rel="stylesheet" href="/StyleSheets/generic-base-1.css"
+type="text/css">
+<link rel="stylesheet" type="text/css" href="../assets/main.css">
+<link rel="shortcut icon" href="/Icons/WWW/Literature.gif">
+<link rel="start" href="../" >
+<style>
+  dt {font-weight: bold; }
+  .timing { font-style: italic }
+   body {counter-reset:section;}
+    h2 {counter-reset:subsection;}
+    h2:before
+    {
+    counter-increment:section;
+    content: counter(section) ". "}
+    h3:before
+    {
+    counter-increment:subsection;
+    content:counter(section) "." counter(subsection) " ";
+    }
+    h4:before
+    {
+    counter-increment:subsubsection;
+  content:counter(section) "." counter(subsection) "." counter(subsubsection) " ";
+    }
+    h4.notoc:before {
+    content: "";
+    }
+</style>
+</head>
+<body>
+
+<div id="header"> <span class="logo"><a href="https://www.w3.org/"><img src="https://www.w3.org/Icons/WWW/w3c_home_nb"
+        alt="W3C"
+        height="48"
+        width="72"></a>
+</span>
+  <div class="breadcrumb"> <a href="https://www.w3.org/participate/">Participate</a> →
+    <h1>How to Close a Working Group</h1>
+  </div>
+  <p class="baseline">This <strong>Guidebook</strong> is the collected
+    wisdom of the W3C Group Chairs, team contacts and other contributors.</p>
+</div>
+
+<h2 id="intro">Introduction</h2>
+
+<p>The W3C Process describes
+the <a href="https://www.w3.org/Consortium/Process/#group-lifecyle">lifecycle
+of chartered groups</a>. 
+At a high level, W3C closes a Working Group
+once it has completed its work.
+</p>
+
+<h2 id="normal">Normal Closure</h2>
+
+<p>
+    Sometimes, after successful completion,
+    the Group is left open to perform <strong>maintenance</strong>;
+    other times, it is closed and a Community Group performs this function.
+</p>
+
+<h2 id="early">Early closure</h2>
+
+<p>Exceptionally, a group may be closed before it has completed its work:</p>
```

Comment:
This seems to be at odds with the W3C Process: the W3C Process identifies causes to clause a group *before the end of its chartered duration*, which is often orthogonal with having *completed its work*.